### ISSUE TITLE
Getting running with Qt Creator

### ISSUE THREAD

**Issue by tomredman** [arBmind/desktop-duplication-cpp#11]
Hello!

I'm trying to get this project up and running using Qt Creator, but I'm having trouble with the build step.

I've tried using the following compilers:
- MinGW
- MSVS C++ Compiler 17.11.35312.102 (x86 - and all the others)

I'm getting a variety of errors depending on which compiler I try. 

Any chance you could provide some Get Started steps with specifics?

![image](https://github.com/user-attachments/assets/cf89205c-b4b9-4c34-888f-7e33e807b407)

![image](https://github.com/user-attachments/assets/1f20f3a1-94be-4e2e-8b35-9ee279bfeafe)

**Comment by arBmind** [comment]
> Any chance you could provide some Get Started steps with specifics?

It seems you already tried a lot. I got the same error while compiling x86. So far I only compiled with amd64.
I pushed a patch to the develop branch that allows building for x86 as well.

On AMD64 the NTAPI is the regular function call. On x86 there is a difference between regular functions and NTAPI functions.
Why do you need an x86 build? It's kind of deprecated for Windows 10/11.

I hope this helps.

**Comment by tomredman** [comment]
Ah, good to know @arBmind! Let me try again with amd64. I assumed I need x86 as I'm on Intel myself, but... very high chance I'm misunderstanding how that works!

**Comment by tomredman** [comment]
Using amd64 worked. Thank you!

**Comment by tomredman** [comment]
For anybody else who sees this, these are the settings that I have working: 
![image](https://github.com/user-attachments/assets/90d9c5c9-2c9d-4857-a8e8-bc256e1e899d)
